paper/paper.md

```diff
@@ -11,36 +11,57 @@ authors:
+# Statement of need
 
-ERAD is designed to be used by researchers, students, community stakeholders, distribution utilities to understand and possibly evaluate effectiveness of different post disaster programs to improve resilience and equity. It was funded by National Renewable Energy Laboratory (NREL) and made publicy available with open license.
+~~In the last few years, we have seen a surge in natural disaster events affecting the power grid, in part due to increased frequency and intensity of extreme weather due to climate change. Similarly, human-driven malicious acts, such as cybersecurity attacks can have similar, wide-spread impacts on the grid. Grid outages in turn effect other services, which have a direct impact on people's lives during an emergency.~~
 
-# Statement of need
+Much of the past work in evaluating equity and resilience have looked at historic data. For instance *<u>(XXX need to add a sentence for each of a 2+ of additional pieces of work, a place to start might be reviewing the citations in [@lin_equity-based_2022])</u>* In another example Brockway, et al. [@brockway_inequitable_2021] show how existing grid infrastructure limits may limit disadvantage community ability to charge electric vehicles and their access to DERs by looking at the geospatial intersection of demographics and grid hosting capacity data. ERAD instead provides a way to look ahead at possible future outages and service access challenges.
+
+Many of the software tools developed for forward-looking understanding of grid resilience have been focused on the bulk power system <u>*(XXX Need to add 2+ citations to support this)*</u>, but this larger scale cannot capture critical service access or outage impacts at the neighborhood, let alone individual customer scale. Existing bulk scale tools, which focus on the integrity of the high-voltage regional, national, and continental scale grid also can not adequately capture the local resilience and equity values of distributed energy resources (DERs) that can provide power to critical service facilities and keep the lights on for customers even when there a challenges getting power from the bulk system. ERAD's enables capturing these local effects at the neighborhood level or even down to the customer level.
```

Comment:
@KapilDuwadi, this is expanded from the comment around bulk systems only in past work. I tried to explain why this matters--since it might not be obvious to many readers, but it still needs a few citations to support the assertion of many past works.